docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        but rather larger jobs which would minimize the effect of the overhead
+        in starting the Kubernetes cluster. Setting
+        <emphasis>useChildProcess</emphasis> to false better allows for the
+        possibility of dynamic scaling. For jobs which would take a long while
+        to compile, the extra (start up) overhead is minimal, and that would
+        be the ideal case to have the <emphasis>useChildProcess</emphasis> as
+        false. Setting <emphasis>useChildProcess</emphasis> to false only
+        allows 1 pod per compile.</para>
+
+        <para><emphasis role="bold">maxActive</emphasis>: The number of how
+        many jobs it can run at any one time. Again use caution because each
+        job will need enough memory to run. For instance, if
+        <emphasis>maxActive</emphasis> is set to 2000, you could submit a very
+        big job and in that case spawn some 2000 jobs using a considerable
+        amount of resources, which could potentially run up a rather expensive
+        compilation bill, again depending on your cloud provider and your
+        billing plan.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Sasha">
+        <title>Sasha</title>
+
+        <para>Sasha is an outlier as it is a dictionary type structure and not
+        a list. You can't have more than one archiver or dfuwu-archiver as
+        that is a value limitation, you can choose to either have the service
+        or not (set the 'disabled' value to true). The Sasha component is an
+        exception to the norm for HPCC Systems components.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Thor">
+        <title>Thor</title>
+
+        <para>In the case of Thor (components) there are similarities with the
+        ECLCC Server. Thor is also a list, and if that list has a value
+        of</para>
+
+        <programlisting>-name: thor </programlisting>
+
+        <para>Or</para>
+
+        <programlisting>-name: thor_50</programlisting>
+
+        <para>If you have a small job you don't need 400 nodes when 10 would
+        do. If you wanted 50 nodes set the number of workers the <emphasis
+        role="bold">numWorkers</emphasis> value to 50 and you would have 50
+        nodes. In a case when multiple Thors would be useful:</para>
+
+        <para><programlisting>thor:
+- name: thor
+  prefix: thor
+  numWorkers: 2</programlisting></para>
+
+        <para>Continuing on this example, if you wanted a 50 node Thor, you
+        set <emphasis>numWorkers:</emphasis> value to 50 and that would deploy
+        a 50 node Thor. In doing so, ideally you should rename the resource to
+        something like <emphasis>thor_50</emphasis> as in the above example,
+        and that would then re-read the file and spawn the 50 node Thor.
+        Caution however, if you were to put that in a helm update that would
+        do nothing, if there are no jobs running there are no Thor instances
+        running.</para>
```

Comment:
this paragraph is misleading.

The reason nothing would happen on a helm update if no jobs are running, is the same reason you don't see 50 pods when you deploy the helm chart in the 1st place.

Thor instances run dynamically. i.e. a 50-way Thor will be spawned when a job is queued to it.
When it is finished that Thor instance will disappear.
This is the same pattern for compile jobs or eclagent jobs, in those cases a single pod per job is spawned (with useChildProcess set to false), in Thor's case, a collection of pods is spawned (manager + N workers).

It should be emphasised that when idle there are no worker (or manager) pods running.
Updating numWorkers will restart the Thor agent listening to the queue, causing all new jobs to use the new configuration.